src/Symfony/Bridge/Monolog/Logger.php

```diff
@@ -27,7 +27,7 @@ class Logger extends BaseLogger implements DebugLoggerInterface
     public function getLogs()
     {
         if ($logger = $this->getDebugLogger()) {
-            return $logger->getLogs();
+            return call_user_func_array(array($logger, 'getLogs'), func_get_args());
```

Comment:
Could be replace with: `$logger->getLogs(...func_get_args());` which is easier to read and faster.

Reply:
not php 5.5 compat

Reply:
Yes, it not supported on php < 5.6 http://sandbox.onlinephpfunctions.com/code/3ba4ee65a58af1267040434818d752f26bc885a1